src/basekernels/matern.jl

```diff
@@ -17,6 +17,10 @@ By default, ``d`` is the Euclidean metric ``d(x, x') = \\|x - x'\\|_2``.
 A Gaussian process with a Matérn kernel is ``\\lceil \\nu \\rceil - 1``-times
 differentiable in the mean-square sense.
 
+!!! note
+
+    Differentiation with respect to the ν parameter is not currently supported.
```

Comment:
I think we should be a bit clearer since we don't plan to change this:
```suggestion
    Differentiation with respect to the order ν is not supported.
```

Reply:
Well, if someone wanted to contribute all the derivatives I'd be happy to include it

Reply:
```suggestion
    Differentiation with respect to the order ν is not currently supported.
```

Reply:
I would more expect request like: when do you plan to support this differentation

Reply:
To a large extent it's also an upstream issue, SpecialFunctions and DiffRules implement the derivatives of the bessel functions but not wrt to the order.